docs/getting_started.md

```diff
@@ -0,0 +1,22 @@
+## How to get
```

Comment:
Add an intro before this, saying what this product is. See https://docs.fetch.ai/CosmPy/ before `To install`. 

Can also reuse some of the text in the previous docs version on keplr here.